sdk/react/panels/cast/CastConnectingScreen.js

```diff
@@ -0,0 +1,103 @@
+import PropTypes from 'prop-types';
+
+import React from 'react';
+import {
+  ActivityIndicator, Animated, Text, TouchableOpacity,
+} from 'react-native';
+
+const Utils = require('../../utils');
+const styles = Utils.getStyles(require('../style/CastConnectingStyles.json'));
+
+export default class CastConnectingScreen extends React.Component {
+  static propTypes = {
+    height: PropTypes.number.isRequired,
+    width: PropTypes.number.isRequired,
+    onDisconnect: PropTypes.func.isRequired,
+  };
+
+  componentWillMount() {
+    const { height } = this.props;
+    this.state = {
+      translateY: new Animated.Value(height),
+      opacity: new Animated.Value(2),
```

Comment:
Don't put functions in the state. Declare it in the constructor and give it more verbose name
```
constructor(props) {
  super(props);
  this.opacityAnimation = new Animated.Value(2);
}
```

Reply:
deleted because of animation isn't used at all